packet/flatcar-linux/kubernetes/cl/controller-install.yaml.tmpl

```diff
@@ -9,6 +9,10 @@ systemd:
         After=network-online.target
         [Service]
         Type=simple
+        Restart=always
+        RestartSec=60
+        StartLimitBurst=5
+        StartLimitIntervalSec=3600
```

Comment:
I think both `StartLimitBurst` and `StartLimitIntervalSec` need to be part of `[Unit]` instead of `[Service]`. 
Ref: https://stackoverflow.com/questions/39284563/how-to-set-up-a-systemd-service-to-retry-5-times-on-a-cycle-of-30-seconds#comment76080028_39284869

Reply:
Fixed.